test/unit/event.js

```diff
@@ -2967,3 +2967,18 @@ if ( !( /firefox/i.test( window.navigator.userAgent ) ) ) {
 		}, 50 );
 	} );
 }
+
+QUnit.test( "contextmenu", function( assert ) {
+	assert.expect( 1 );
+
+	var times = 0,
+	    handler = function() { ++times; };
+
+	jQuery( "#firstp" )
+	    .contextmenu( handler )
+	    .contextmenu()
```

Comment:
We actually already have [a test for adding and triggering "contextmenu"](https://github.com/jquery/jquery/blob/master/test/unit/event.js#L2861). @markelog What tests were you thinking of? Ensuring the event object is fixed for contextmenu?